test/Spec.hs

```diff
@@ -107,15 +105,13 @@ tests docker0 = testGroup "Tests"
         $ expectFailure $ withMaxSuccess 1000 $ prop_parallel_clean     (Eq True)  Cleanup.NoBug     NoOp
 
       , testProperty "3-threadsRegularNoOp"   $ prop_nparallel_clean  3 Regular    Cleanup.NoBug     NoOp
-      , testProperty "3-threadsRegular"
-        $ expectFailure                       $ prop_nparallel_clean  3 Regular    Cleanup.NoBug     ReDo
+      , testProperty "3-threadsRegular"       $ prop_nparallel_clean  3 Regular    Cleanup.NoBug     ReDo
       , testProperty "3-threadsRegularExc"    $ expectFailure
                                               $ prop_nparallel_clean  3 Regular    Cleanup.Exception NoOp
       , testProperty "3-threadsRegularExc"
         $ expectFailure                       $ prop_nparallel_clean  3 Regular    Cleanup.Exception ReDo
       , testProperty "3-threadsFilesNoOp"     $ prop_nparallel_clean  3 Files      Cleanup.NoBug     NoOp
-      , testProperty "3-threadsFiles"
-        $ expectFailure                       $ prop_nparallel_clean  3 Files      Cleanup.NoBug     ReDo
+      , testProperty "3-threadsFiles"         $ prop_nparallel_clean  3 Files      Cleanup.NoBug     ReDo
```

Comment:
Note these two tests no longer fail. It seems intuitive to think that if the sequential version of them doesn't fail these should also not fail (see above, `seqRegular` and `seqFiles` are not expected to fail). It seems this failure was artificial induced by how the library used to do things before this change?

Reply:
> Note these two tests no longer fail. It seems intuitive to think that if the sequential version of them doesn't fail these should also not fail (see above, seqRegular and seqFiles are not expected to fail).

I don't remember why these tests were added with `expectFailure` if they indeed are supposed to pass, is there anything documented about this in the commit logs?

>  It seems this failure was artificial induced by how the library used to do things before this change?

Would be good to understand what exactly changed to cause them to pass now though?

Reply:
Lol it seems it was me the one that added those when I provided the WithSetup variants. So probably I broke it then :(

Reply:
Ok, good!